### PR TITLE
packages: rename @backstage/app to example-app

### PR DESCRIPTION
**Pull request by Rugvip**
Making it a bit more clear that it's just an example and not a published package

### CODE REVIEW

File: packages/app/package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@backstage/app",
+  "name": "example-app",
```
Comment (freben) [inline packages/app/package.json]: but you didn't rename the actual directory?

Comment (Rugvip) [reply packages/app/package.json]: Yeah no, wondering if we should

Comment (freben) [reply packages/app/package.json]: mm, i am not entirely sure but thought that maybe that communicates the purpose even more strongly than the package name ... the directory is the first thing you see

Comment (Rugvip) [reply packages/app/package.json]: can ship this though and discuss that in a separate PR?

Comment (freben) [reply packages/app/package.json]: sure.

Comment (Rugvip) [reply packages/app/package.json]: approve? :grin:

**Review by soapraj — APPROVED** [review]

**Pull request merged**